Content.Server/Cargo/Systems/CargoSystem.Shuttle.cs

```diff
@@ -356,6 +371,7 @@ private void SendToCargoMap(EntityUid uid, CargoShuttleComponent? component = nu
 
         UpdateShuttleCargoConsoles(component);
         _sawmill.Info($"Stashed cargo shuttle {ToPrettyString(uid)}");
+        ReportDepartingItems(component);
     }
```

Comment:
This doesn't fix the issue because it only handles 1 of many different map changes.

Reply:
I think the best thing to do for now is just iterate active magnets and check that the target is on the same map.

Reply:
> I think the best thing to do for now is just iterate active magnets and check that the target is on the same map.

Is it okay to put the check in Update, running every frame?

Reply:
Yeah, there's only 1 magnet most of the time and I think the other alternatives might be insufficient (e.g. subbing to grid map changes or re-adding the map change event back).